test-suite/Makefile

```diff
@@ -242,6 +242,7 @@ $(foreach S,$(VSUBSYSTEMS),$(eval $(call vdeps,$(S))))
 #######################################################################
 
 summary_dir = echo $(1); find $(2) -name '*.log' -print0 | xargs -0 tail -q -n1 | sort
```

Comment:
would this work?
```suggestion
# some versions of tail do not accept more than ~1k arguments
summary_dir = echo $(1); find $(2) -name '*.log' -print0 | xargs -0 -L 999  tail -q -n1 | sort
```
(and no separate summary_dir_bug)

Reply:
Yes, it works. Much better. Pushed.